### DISCUSSION TITLE
AI Villagers and golems

### DISCUSSION THREAD

**Discussion by Nyasha1234** [Roadhog360/Et-Futurum-Requiem#453] [Ideas]
Github deactivated my other account for some reason unknown to me. My topic here has also been deleted. Therefore, I will create this theme again. If you deleted it, please let me know. In the meantime, I'll post the deleted text. 

It would be cool to move content related to AI Villagers and goes to 1.7.10, as well as trading from 1.14. Given that in new versions of Minecraft trading with Villagers breaks the balance of the game, it may make sense to make this trade not so cheating. Integration with Village Name will look very cool.
Thank you for your hard work! Your mod is great.

I've seen your answer and really hope you can add this.

**Comment by Roadhog360** [comment]
I've thought about having an option for seeded trade rerolling, so you can't cheese villagers and would discourage trading halls.

**Comment by Darek505** [comment]
The question is a little off topic, but still: is it possible to consider adding integration with CraftTweaker / custom json to be able to change trades? The problem is that in most modpacks, villagers are simply useless, as they sell things that are absolutely useless in 90% of cases. And there is no mod that would do this (with the exception of “custom villagers” by Elec332, but it just random crashes).
 So I think the ability to create your own trades via CraftTweaker/json will be incredibly useful (but this is just a suggestion)

**Reply by Roadhog360** [reply]
This would be suited for a mod other than Et Futurum Requiem.
